pkgs/by-name/si/silverbullet/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  deno,
+}:
```

```suggestion
{ lib
, stdenv
, fetchurl
, deno
}:
```

this is the conventional formatting for nixpkgs

---

thanks, I used `nixfmt-rfc-style` too soon